### ISSUE TITLE
CertificateValidationRemoteServer_EndToEnd_Ok fails on OS X

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#5555]
```
09:41:41    System.Net.Security.Tests.CertificateValidationRemoteServer.CertificateValidationRemoteServer_EndToEnd_Ok [FAIL]
09:41:41       Assert.Equal() Failure
09:41:41       Expected: None
09:41:41       Actual:   RemoteCertificateChainErrors
09:41:41       Stack Trace:
09:41:42             at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
09:41:42             at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
09:41:42             at System.Net.Security.SslStream.EndAuthenticateAsClient(IAsyncResult asyncResult)
09:41:42             at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
09:41:42          --- End of stack trace from previous location where exception was thrown ---
09:41:42             at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
09:41:42             at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
09:41:42             at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
09:41:42             at System.Net.Security.Tests.CertificateValidationRemoteServer.<CertificateValidationRemoteServer_EndToEnd_Ok>d__0.MoveNext()
09:41:42          --- End of stack trace from previous location where exception was thrown ---
09:41:42             at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
09:41:42             at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
09:41:42          --- End of stack trace from previous location where exception was thrown ---
09:41:42             at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
09:41:42             at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
09:41:42          --- End of stack trace from previous location where exception was thrown ---
09:41:42             at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
09:41:42             at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
```
It passes on Ubuntu.

**Comment by eerhardt** [comment]
I see this on my OSX 10.11 machine as well.

**Comment by bartonjs** [comment]
OSX: The problem is the root store is, by default, empty.
CentOS: The problem is the root store is compressed into a single file, and we only read the first certificate from it.

The fix I was planning here is to try to make the test resilient to not having root trust; but the real CentOS fix might be to multi-read that file.

**Comment by bartonjs** [comment]
The CentOS fix is in, and the test was re-enabled (#5765).  OS X was still failing, but that should be fixed by #5779.  Once today's build builds, makes it out to myget, and the test bundle gets updated, we can completely remove the [ActiveIssue] marker and close this issue.
